extensions/roc-plugin-style-css/src/css/pipeline.js

```diff
@@ -19,12 +24,17 @@ export default function cssPipeline(base, loaders, isDist, sourceMap = false, cs
         `!${loaders.join('!')}` :
         '';
 
+    const preLoaders = (ploaders && ploaders.length > 0) ?
+        `!${ploaders.join('!')}` :
+        '';
+
     // We set importLoaders to nr. loaders + 1 to get css-loader to process everything through the pipeline
     return `${require.resolve(base)}?` +
         `${sourceMapSettings}` +
         '-autoprefixer&' +
         `importLoaders=${loaders.length + 1}` +
```

Comment:
I think we should adjust the `importLoaders` value here to take preLoaders into consideration.